exercises/robot-simulator/robot_simulator_test.cpp

```diff
@@ -0,0 +1,366 @@
+#define BOOST_TEST_MAIN
+#include <boost/test/unit_test.hpp>
+#include "robot_simulator.h"
+
+// Robot-Simulator exercise test case data version 2.0.0
+
+BOOST_AUTO_TEST_CASE(Robots_are_created_with_a_position_and_direction)
+{
+    const robot_simulator::Robot r;
```

Comment:
This file is long enough I'd add a using declaration for both `Robot` and `Bearing` just to make it a little shorter to read.

Reply:
You're right. This is saving a lot of text. So, I added these two using declarations, as you're suggesting.